stingray/tests/test_largememory.py

```diff
@@ -0,0 +1,97 @@
+import tempfile
+
+import pytest
+import numpy as np
+from stingray.lightcurve import Lightcurve
+from stingray.events import EventList
+from stingray.largememory import createChunkedSpectra, saveData
+from stingray.crossspectrum import AveragedCrossspectrum
+from stingray.powerspectrum import AveragedPowerspectrum
+
+
+HAS_ZARR = False
+try:
+    import zarr
+
+    HAS_ZARR = True
+    from numcodecs import Blosc
+except ImportError:
+    pass
+
+
+class TestSaveAndRetrieve(object):
+    @classmethod
+    def setup_class(cls):
+        time = np.arange(0, 1e7)
+        counts = np.random.poisson(10, time.size)
+        cls.lc = Lightcurve(time, counts, skip_checks=True)
+        evtimes = np.sort(np.random.uniform(0, 1e7, 10**7))
+        cls.ev = EventList(time=evtimes)
+
+    @pytest.mark.skipif('not HAS_ZARR')
+    def test_save_wrong_data(self):
+        with pytest.raises(ValueError) as excinfo:
+            saveData("A string", 'bububu')
+        assert 'Cannot save data of type str' in str(excinfo.value)
+
+    @pytest.mark.skipif('not HAS_ZARR')
+    def test_save_lc(self):
+        file = tempfile.mkdtemp()
+        saveData(self.lc, file)
+        newfile = zarr.open(file, mode='r')
+        # TODO: Retrieve the data and test that they are identical to the
```

Comment:
Please complete all the TODOs here for proper testing.
Once you retrieve the data into a new Eventlist or Lightcurve, please test that all attributes are identical to the original ones